src/routes/articles.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import ArticleController from '../controllers/article';
+import verifySlug from '../middlewares/verifySlug';
+import { verifyToken, checkUser } from '../middlewares/authentication';
+import { checkInput } from '../middlewares/validateArticle';
+import { paramsChecker } from '../middlewares/checkParams';
+
+const articlesRouter = express.Router();
+
+const {
+  createArticle,
+  getAllArticles,
+  updateArticle,
+  getArticle,
+  deleteArticle
+} = ArticleController;
+
+const { slugChecker } = verifySlug;
+
+articlesRouter.get('/articles', verifyToken, getAllArticles);
+articlesRouter.get('/articles/:slug', verifyToken, slugChecker, getArticle);
+articlesRouter.post('/articles/:userId', verifyToken, paramsChecker, checkInput, createArticle);
```

Comment:
How come the `:userId` parameter is part of the route path for this and the next two endpoints?

Here are shots of the API specs again for reference:

<img width="932" alt="screen shot 2018-12-04 at 6 50 29 am" src="https://user-images.githubusercontent.com/15332525/49421631-0621d800-f791-11e8-9fdd-b74e20bc24e0.png">
<img width="952" alt="screen shot 2018-12-04 at 6 50 16 am" src="https://user-images.githubusercontent.com/15332525/49421632-0621d800-f791-11e8-926a-17e8e3ba0452.png">
<img width="270" alt="screen shot 2018-12-04 at 6 50 43 am" src="https://user-images.githubusercontent.com/15332525/49421633-06ba6e80-f791-11e8-8fef-e13b1f34fd21.png">